sdk/client/src/client.rs

```diff
@@ -284,6 +286,7 @@ impl Client {
             .inner
             .post(&self.url)
             .json(payload)
+            .header(reqwest::header::USER_AGENT, USER_AGENT)
```

Comment:
Its preferable to keep using this provided header in the async client case though